tests/02_eradiate/01_unit/scenes/phase/test_tabulated.py

```diff
@@ -8,17 +8,28 @@
 
 
 @pytest.fixture
-def dataset():
+def regular_mu() -> xr.DataArray:
+    """Phase function data with a regular mu grid."""
     result = xr.open_dataset(
-        path_resolver.resolve("tests/radprops/rtmom_aeronet_desert.nc")
+        path_resolver.resolve("tests/spectra/particles/random-regular_mu.nc")
     ).load()
     result.close()
-    return result
+    return result.phase
 
 
-def test_tabulated_basic(modes_all, dataset):
+@pytest.fixture
+def irregular_mu() -> xr.DataArray:
+    """Phase function data with an irregular mu grid."""
+    result = xr.open_dataset(
+        path_resolver.resolve("tests/spectra/particles/random-irregular_mu.nc")
+    ).load()
```

Comment:
Getting rid of the path resolver would be nice. Two possibilities:

* use `data.load_dataset()` (this data should be registered to the data store), or
* create it as a fixture, preferable if the data is used only in this test.